cell.py

```diff
@@ -1,8 +1,41 @@
+from collections.abc import Iterable
+
+
 class Cell:
-    # TODO: implement
 
-    def __init__(self):
-        self.alternatives = {1, 2, 3, 4, 5, 6, 7, 8, 9}
+   
+    def __init__(self,init_value):
```

Comment:
стоит добавить значение по дефолту
```suggestion
    def __init__(self, init_value = None):
```